### PR TITLE
+glab

### PR DESCRIPTION
**Pull request by ddobrinskiy**

### CODE REVIEW

**Comment by ddobrinskiy** [comment]
Can't get it to show the correct version instead of DEV, not sure where the problem is

**Comment by ddobrinskiy** [comment]
@mxcl almost done, would appreciate your help with one last thing: can't get it to show the correct version instead of DEV, not sure where the problem is

File: projects/gitlab.com/gitlab-org/cli/package.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    go mod download
+    mkdir -p "{{ prefix }}"/bin
+    go build -v -trimpath -ldflags="$LDFLAGS" -o $BUILDLOC ./cmd/glab #FIXME for some reason glab --version returns DEV instead of 1.26.0
+  dependencies:
+    go.dev: ^1.18
+  env:
+    GO111MODULE: 'on'
+    BUILDLOC: '{{prefix}}/bin/glab'
+    LDFLAGS:
+      [-s, -w, "-X=main.Version={{version}}", "-X main.debugMode=false"]
```
Comment (mxcl) [inline projects/gitlab.com/gitlab-org/cli/package.yml]: It'll be this, it varies by go pkg (lol) you may have to look at their CI or look into the source to figure out what key is correct.

Comment (jhheider) [reply projects/gitlab.com/gitlab-org/cli/package.yml]: Hmmmmm https://gitlab.com/gitlab-org/cli/-/blob/main/.goreleaser.yml#L19-20

Comment (jhheider) [reply projects/gitlab.com/gitlab-org/cli/package.yml]: https://gitlab.com/gitlab-org/cli/-/blob/main/Makefile#L50 >
https://gitlab.com/gitlab-org/cli/-/blob/main/Makefile#L16

Maybe: `build.env.GLAB_VERSION: ${{ version }}`?

**Comment by jhheider** [comment]
Good work, @ddobrinskiy . Thanks!

**Pull request merged**